src/app/visual/colorpicker/colorpicker-visual.component.html

```diff
@@ -2,26 +2,39 @@
   (themeSettingsChange)="themeSettingsChange($event)"
 ></sky-docs-demo-control-panel-theme>
 
-<div
-  id="screenshot-colorpicker"
+<button
+  class="sky-btn sky-btn-default"
+  id="screenshot-action-button-center-align"
+  type="button"
+  (click)="onToggleAbleColorpicker()"
 >
-  <sky-colorpicker
-    labelledBy="FOOBAR"
-    #colorPickerExample1
+  Toggle Colorpicker #1's Ability
+</button>
+
+<form
+  [formGroup]="myForm"
+>
+  <div
+    id="screenshot-colorpicker"
   >
-    <input
-      [initialColor]="selectedColor1"
-      [outputFormat]="selectedOutputFormat1"
-      [presetColors]="presetColors1"
-      [skyColorpickerInput]="colorPickerExample1"
-      [(ngModel)]="color1"
+    <sky-colorpicker
+      labelledBy="FOOBAR"
```

Comment:
`labelledBy` pointing to an invalid id. Maybe we could just remove this?

Reply:
done in last commit